molecule/debian/molecule.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# This molecule configuration file is suitable for testing Ansible
```

Comment:
If this isn't going to symlink to `molecule-no-systemd.yml` or `molecule-with-systemd.yml`, does it makes sense to have those files symlinked here?

Reply:
The point is to make the non-`default` scenarios look as much like the usual `default` scenario as possible.  That way it's easy to see where everything came from.  Also, if `molecule-no-systemd.yml` changes in the upstream skeleton then the changes are more likely to get noticed and moved into `debian/molecule.yml` this way.

Reply:
That's reasonable. I just wanted to make sure we had intent behind leaving them. Thanks for the reply 👍